### ISSUE TITLE
Wrong rotation of LocationLayer inside Drop-in UI

### ISSUE THREAD

**Issue by P-Zenker** [mapbox/mapbox-navigation-android#256]
Hey,

I've downloaded your [sample project](https://github.com/mapbox/mapbox-navigation-android/tree/master/navigation/app/src/main/java/com/mapbox/services/android/navigation/testapp/activity) today. 

I don't know if this issue belongs here or rather to mapbox-plugins-android. I think the rotion should be counter-clockwise instead of clockwise in such cases, where the rotation angle is > 180 degree.

![2017-09-21 14 40 06](https://user-images.githubusercontent.com/23097192/30696179-0b14481c-9edb-11e7-91f4-7b29a4894142.gif)

My route was from lat/lng 53.867324, 10.673620 to lat/lng 53.866998, 10.680687.

Best regards,
Patrick

**Comment by mf-mindcrewtech** [comment]
Same issue with me, @P-Zenker did you find any solution?

**Comment by danesfeder** [comment]
@P-Zenker @mf-mindcrewtech Are you able to reproduce this in `0.6.0`

**Comment by P-Zenker** [comment]
No, I am not. With 0.6.0 the rotation for this one case is as expected (correct).

Setup (with identical lat / lng):
```
    compile('com.mapbox.mapboxsdk:mapbox-android-sdk:5.1.4@aar') {
        transitive = true
    }
    compile 'com.mapbox.mapboxsdk:mapbox-android-navigation:0.6.0'
    compile('com.mapbox.mapboxsdk:mapbox-android-navigation-ui:0.6.0') {
        transitive = false
    }
```

**Comment by danesfeder** [comment]
@P-Zenker Nice, thank you - also, when you compile: 
```
compile('com.mapbox.mapboxsdk:mapbox-android-navigation-ui:0.6.0') {
     transitive = true
}
```
^ you will get the Map SDK and the core navigation logic, so you don't need to compile the other two dependencies

**Comment by cammace** [comment]
I've opened https://github.com/mapbox/mapbox-navigation-android/issues/316 which is a similar issue with the camera, will be looking into this further in the coming weeks. For now, it sounds like this issues been resolved in 0.6.0. Closing.
